ui/components/app/advanced-gas-fee-popover/advanced-gas-fee-gas-limit/advanced-gas-fee-gas-limit.test.js

```diff
@@ -65,4 +65,46 @@ describe('AdvancedGasFeeGasLimit', () => {
     fireEvent.click(screen.queryByText('Edit'));
     expect(document.getElementsByTagName('input')[0]).toHaveValue(21000);
   });
+
+  it('should show error if gas limit is not in range', () => {
+    render();
+    fireEvent.click(screen.queryByText('Edit'));
+    fireEvent.change(document.getElementsByTagName('input')[0], {
+      target: { value: 20000 },
+    });
+    expect(
+      screen.queryByText(
+        'Gas limit must be greater than 20999 and less than 7920027',
+      ),
+    ).toBeInTheDocument();
+    fireEvent.change(document.getElementsByTagName('input')[0], {
+      target: { value: 8000000 },
+    });
+    expect(
+      screen.queryByText(
+        'Gas limit must be greater than 20999 and less than 7920027',
+      ),
+    ).toBeInTheDocument();
+    fireEvent.change(document.getElementsByTagName('input')[0], {
+      target: { value: 7000000 },
+    });
+    expect(
+      screen.queryByText(
+        'Gas limit must be greater than 20999 and less than 7920027',
+      ),
+    ).not.toBeInTheDocument();
+  });
+
+  it('should validate gas limit against minimumGasLimit it is passed to context', () => {
+    render({ minimumGasLimit: '0x7530' });
```

Comment:
Does this one need to change now too?
```suggestion
    render({ minimumGasLimitDec: '0x7530' });
```